components/tag/basic/src/index.scss

```diff
@@ -0,0 +1,51 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+
+.sui-TagBasic {
```

Comment:
I'm using hardcoded values until theme-basic is migrated

Reply:
Ok, I understand this point, but if we leave it like this other verticals won't be able to reuse this component, just with an ugly practice: overwritting variables. I'd rather create some default variables for those values within this file or in a specific place inside the theme-basic.

Reply:
Yes, of course, I'm waiting for the final name and value of most of them in order to update this file